.github/workflows/continuous-monitoring.yml

```diff
@@ -27,6 +27,14 @@ jobs:
                aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
                aws-region: us-east-1
 
+          - uses: actions/setup-dotnet@v1
```

Comment:
Does this make more sense?

https://github.com/actions/setup-dotnet

```
with:
    dotnet-version: | 
      3.1.x
      6.0.x
```

Reply:
Yes. Indeed. 🙂